src/main/java/org/mockito/internal/matchers/Any.java

```diff
@@ -10,7 +10,7 @@
 import org.mockito.ArgumentMatcher;
 
 @SuppressWarnings("unchecked")
-public class Any implements ArgumentMatcher, Serializable {
```

Comment:
Hmm, i am having a bad feeling about changing code that is not related to the solution of the issue. This change should be part of an other pull request. WDYT?